docs/install.rst

```diff
@@ -41,17 +41,19 @@ The easiest way to install Python and SciPy is with `Anaconda`_, a free scientif
 
     jupyter notebook
 
-Note that if you are used to starting Jupyter notebook from the Anaconda Navigator, you will have to set the environment to `pylake` for it to have access to the environment where you just installed Pylake.
-You can do this in the drop down menu that normally defaults to the environment `base`.
+.. note::
 
-If you are used to starting Jupyter from the Anaconda prompt, then remember to activate the correct environment (pylake) prior to starting the notebook.
-You can activate this environment by calling::
+    If you start Jupyter from the Anaconda prompt, then remember to activate the correct environment (`pylake`) prior to starting the notebook. You can activate this environment by invoking `conda activate pylake` in that command prompt before doing anything else.
 
-    conda activate pylake
+.. note::
+
+    When starting Jupyter notebook from the Anaconda Navigator, first select the environment you installed Pylake in from the "Applications on" drop down menu (if you followed the above instructions, this environment will be named `pylake`). This drop down menu normally defaults to an environment named `base`.
 
```

Comment:
Nice addition, hopefully this will reduce the nr of questions about installation.

Very minor comment: aesthetically, I'd prefer one note, rather than 2. This note could say something like

Make sure to activate your environment
In the command line this is done by...
In the Anaconda Navigator this is done by....